Test/test_installation/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ set_property(TARGET fmilib PROPERTY IMPORTED_LOCATION "${FMIL_HOME}/lib/${CMAKE_
 # Build the test
 add_executable(main main.c)
 if(WIN32)
-    target_link_libraries(main PRIVATE fmilib Shlwapi)
+    target_link_libraries(main PRIVATE fmilib shlwapi)
```

Comment:
Can you remind me why was this changed? I felt like I saw this change recently in another PR too but now I am not sure.

Rest of the changes look good!

Reply:
its needed when compiling on a case-sensitive fs like with linux/mingw